File: app/src/main/java/com/example/bakerystore/models/OrderDetailResponse.kt
package com.example.bakerystore.models

data class OrderDetailResponse(
    val orderId: Int,
    val userId: Int,
    val totalAmount: Double,
    val status: String?,
    val deliveryDate: String?,
    val createdAt: String?,
    val paymentMethod: String?,
    val paymentStatus: String?,
    val shippingAddress: String?,
    val items: List<OrderDetailItemResponse>
)

File: backend/BakeryStoreApi/DTOs/OrderDetailResponse.cs
namespace BakeryStoreApi.DTOs
{
    public class OrderDetailResponse
    {
        public int OrderId { get; set; }

        public int UserId { get; set; }

        public decimal TotalAmount { get; set; }

        public string? Status { get; set; }

        public DateTime? DeliveryDate { get; set; }

        public DateTime CreatedAt { get; set; }

        public string? PaymentMethod { get; set; }

        public string? PaymentStatus { get; set; }

        public string? ShippingAddress { get; set; }

        public List<OrderDetailItemResponse> Items { get; set; } = new List<OrderDetailItemResponse>();
    }
}